sql/files/defaultdata/compare/compare.cc

```diff
@@ -176,16 +153,46 @@ int main(int argc, char **argv) {
 			if (!isfloat(team.c_str(), tval)) {
```

Comment:
I think you can also take this out as you have: https://github.com/DOMjudge/domjudge/pull/1863/files#diff-30594013bdc266f0b1b51ef87283423017bce078812a75f1c00a6a0716e1b7adR186

Reply:
This `isfloat` is needed to populate `tval`, which is of type `flt`. Without it, the `isfinite`/`isinf`/`isnan` functions don't have a value for `tval` :slightly_smiling_face: (this should be clearer now that I've re-extracted the `compare_float` function)

Reply:
Check!